### ISSUE TITLE
Focus doesn't go to top of the screen/page for "Now you can upload your codes" with VoiceOver on [Global issue]

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#507]
With VoiceOver on, when the user goes to ""Now you can upload your codes"", the focus is on last paragraph/sentence. It should be on top of the page. Tried it twice and the focus always goes to this area.

Screenshot below shows where the focus land for the VoiceOver user.

iOS 13.5.1 on iPhone 7

![image](https://user-images.githubusercontent.com/14280062/86840064-44b7fc80-c070-11ea-8efc-7dfe058692dd.png)

**Comment by deandreleach** [comment]
#490 #521 #519 [GitHub](https://github.com/hakimel/Avgrund/blame/3ae7c75e6dbaa8ab6458e3b32bd03e4e955f7763/multiple.html#L8)

**Comment by paulettefrancis** [comment]
With VoiceOver on, when the user goes to "Now you can upload your codes" page, the focus now appears on the previous page and not on the header of the current page.

Tested on iPhone XR 13.5.1

![Focus](https://user-images.githubusercontent.com/22638158/87703361-b840c480-c768-11ea-80db-51e80ffdac48.jpg)
